packages/react-badge/src/components/CounterBadge/__snapshots__/CounterBadge.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CounterBadge renders a default state 1`] = `
+<div
+  className=" "
+>
+  undefined
```

Comment:
This `undefined` looks like a bug?

Reply:
Ah, it's probably because in `useCounterBadge` you didn't provide fallback values when converting count to a string `${}`:
```typescript
(props.count || count) > (props.overflowCount || overflowCount) ? `${props.count}+` : `${props.count}`,
```

That should be fixed with my other suggestion in `useCounterBadge`